internal/utils/helpers.go

```diff
@@ -73,6 +75,16 @@ func CreateWorkDir(fs afero.Fs) (string, error) {
 	return workDir, nil
 }
 
+// CleanupWorkDir removes all files in a directory
+// Return nothing so we can use it with defer more conveniently
+func CleanupWorkDir(fs afero.Fs, path string) {
+	err := fs.RemoveAll(path)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		os.Exit(1) // I guess..?
+	}
+}
```

Comment:
Why not:

```suggestion
func CleanupWorkDir(fs afero.Fs, path string) error {
	return fs.RemoveAll(path)
}
```

Or even:
```suggestion
func CleanupWorkDir(fs afero.Fs, path string) {
	_ := fs.RemoveAll(path)
}
```

Reply:
Any returns from defer functions are [ignored](https://go.dev/ref/spec#Defer_statements)

Reply:
But I think maybe I get a lint-fix error for ignoring an error.